### DISCUSSION TITLE
Failing inverter was changed, how to refresh inverter list ?

### DISCUSSION THREAD

**Discussion by gbetous** [ksheumaker/homeassistant-apsystems_ecur#252] [General]
Hi,

An inverter was failing and have been replaced by the fitter. Everything is in place and now appears on the AP System's application. I'd like to update also in HomeAssistant, but I don't want to loose all the history of my production.

How can I do this ?

Thanks very much !

**Comment by gbetous** [comment]
My bad, I did not even notice that they were automagically detected.

Really, this is an amazing integration, thank you very much !!!

**Reply by HAEdwin** [reply]
If you haven't allready removed the failing inverter, instructions are here: https://github.com/ksheumaker/homeassistant-apsystems_ecur?tab=readme-ov-file#removing-devices
